src/widgets/dock_area/mod.rs

```diff
@@ -176,14 +176,16 @@ impl<'tree, Tab> DockArea<'tree, Tab> {
         self.allocate_area_for_root(ui);
 
         for node_index in (0..self.tree.len()).map(NodeIndex) {
-            if self.tree[node_index].is_parent() {
-                self.split_subtree(ui, node_index);
+            if self.tree[node_index].is_leaf() {
+                self.show_leaf(ui, &mut state, node_index, tab_viewer);
             }
         }
 
+        // Draw separator after node UI so that we can overlay the separator
```

Comment:
Are you sure this reordering is a good idea?

I remember ordering these loops this way in order to fix a bug where separators sometimes didn't respond to interactions when you moved the tabs around. See: #3

Reply:
I looked at the issue you mentioned and played around with tabs but never stumbled into the problem, so I think it is fine. The crate changed a lot since then so maybe the original cause of the issue doesn't exists anymore?

Since you're more familiar with the issue maybe you could try to reproduce on your side just to be sure?

The reordering is mandatory for the patch to work, I tried other things but for the allocated `interact_rect` to be reactive to clicks it must be drawn above (after) the underlying UI.

Reply:
Works alright, actually :+1: